### PR TITLE
A4A Examples for Banner and Carousel

### PR DESCRIPTION
**Pull request by carlosfuertes**
Examples of A4A for banner and carousel (slide-show style).

### CODE REVIEW

File: src/60_Sample_A4As/A4A_Banner_Ad.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!--
+  #### Introduction
+
+  This is a simple banner [AMP Ad for AMP Pages](https://github.com/ampproject/amphtml/issues/3133) (A4A). In general an A4A should be a valid AMP document.
```
Comment (sebastianbenz) [inline src/60_Sample_A4As/A4A_Banner_Ad.html]: With the new ABE design we show the first sample sentence in a card on the homepage. Suggestion: rephrase to "This samples demonstrates how to implement a simple banner [AMP Ad for AMP Pages](https://github.com/ampproject/amphtml/issues/3133) (A4A)."

File: src/60_Sample_A4As/A4A_Banner_Ad.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  <a target="_blank"
+     href="https://ampbyexample.com/sample_a4as/a4a_banner_ad/">
+     <amp-img src="/img/amp_300x250.jpg"
+              width=300
+              height=250
+              alt="a4a image"/>
+  </a>
+
+  <!--
+   We include a tracking pixel to record that the ad was rendered using [amp-pixel](https://github.com/ampproject/amphtml/blob/master/builtins/amp-pixel.md).
```
Comment (sebastianbenz) [inline src/60_Sample_A4As/A4A_Banner_Ad.html]: Should we mention that amp-analytics would work as well for tracking?

File: src/60_Sample_A4As/A4A_Carousel.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+  We use type="slides" to display a list of nodes as slides: Each of these nodes may have arbitrary AMP-HTML children. Here we put in every of them a simple banner ad.
+
+  See [amp-carousel](https://ampbyexample.com/components/amp-carousel/) for more options to display the content.
+  -->
+  <amp-carousel width="300"
```
Comment (sebastianbenz) [inline src/60_Sample_A4As/A4A_Carousel.html]: suggestion: give the slides a different background color

**Comment by sebastianbenz** [comment]
Looks good! Added a few comments. Also:

* suggestion: rename category to Ads for AMP (A4A)
* create a index.json with a short description of the new category ([sample](/src/30_Advanced/index.json)) (we just released this feature today)

File: src/60_Sample_A4As/A4A_Banner_Ad.html (hunk 1 above)
Comment (carlosfuertes) [inline src/60_Sample_A4As/A4A_Banner_Ad.html]: Done.

File: src/60_Sample_A4As/A4A_Banner_Ad.html (hunk 2 above)
Comment (carlosfuertes) [inline src/60_Sample_A4As/A4A_Banner_Ad.html]: Done.

File: src/60_Sample_A4As/A4A_Carousel.html (hunk 3 above)
Comment (carlosfuertes) [inline src/60_Sample_A4As/A4A_Carousel.html]: Done.

**Comment by carlosfuertes** [comment]
Addressed comments, renamed category and created corresponding index.json. Thanks!

**Comment by sebastianbenz** [comment]
Can you mark the samples as drafts (this hides them from the frontpage, but makes them accessible via direct links). Otherwise LGTM.

For draft mode add the following comment to the samples files: 

```
<!---{
 "draft": true
}--->
```

**Comment by carlosfuertes** [comment]
Marked the samples as drafts.

**Comment by sebastianbenz** [comment]
LGTM

**Pull request merged**